airbyte-proxy/test.sh

```diff
@@ -5,13 +5,14 @@ PORT=18000
 BASIC_AUTH_USERNAME=airbyte
 BASIC_AUTH_PASSWORD=password
 BASIC_AUTH_UPDATED_PASSWORD=pa55w0rd
+BASIC_AUTH_PROXY_TIMEOUT=120
 TEST_HOST=localhost
 VERSION="${VERSION:-dev}" # defaults to "dev", otherwise it is set by environment's $VERSION
 
 echo "testing with proxy container airbyte/proxy:$VERSION"
 
 function start_container () {
-  CMD="docker run -d -p $PORT:8000 --env BASIC_AUTH_USERNAME=$1 --env BASIC_AUTH_PASSWORD=$2 --env PROXY_PASS_WEB=http://localhost --env PROXY_PASS_API=http://localhost --name $NAME airbyte/proxy:$VERSION"
+  CMD="docker run -d -p $PORT:8000 --env BASIC_AUTH_USERNAME=$1 --env BASIC_AUTH_PASSWORD=$2 --env BASIC_AUTH_PROXY_TIMEOUT=$3 --env PROXY_PASS_WEB=http://localhost --env PROXY_PASS_API=http://localhost --name $NAME airbyte/proxy:$VERSION"
```

Comment:
Just tried, unfortunately if I don't provide the variable the test fails, because I am trying to define a timeout of inexistent value. I could define a default value on `run.sh`, but I don't know if that is the best solution

Reply:
Ah ha! That makes sense - normal usage will get the value from `.env`, but not in the test.  This is OK then!